.github/workflows/cypress_workflow.yml

```diff
@@ -42,18 +40,65 @@ env:
   COMMENT_TAG: '[MANUAL CYPRESS TEST RUN RESULTS]'
   COMMENT_SUCCESS_MSG: ':white_check_mark: Cypress test run succeeded!'
   COMMENT_FAILURE_MSG: ':x: Cypress test run failed!'
+  LATEST_VERSION: '2.17.0'
 
 jobs:
   cypress-tests:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-          group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        include:
+          # Standard test configs (groups 1-9)
+          - group: 1
+            config: standard
+            test_location: ftr
+          - group: 2
+            config: standard
+            test_location: ftr
+          - group: 3
+            config: standard
+            test_location: ftr
+          - group: 4
+            config: standard
+            test_location: ftr
+          - group: 5
+            config: standard
+            test_location: ftr
+          - group: 6
+            config: standard
+            test_location: ftr
+          - group: 7
+            config: standard
+            test_location: ftr
+          - group: 8
+            config: standard
+            test_location: ftr
+          - group: 9
+            config: standard
+            test_location: ftr
+          # Query enhanced tests
+          - group: 10
+            config: query_enhanced
+            test_location: ftr
+          # Dashboard tests
+          - group: 11
+            config: dashboard
+            test_location: source
     container:
       image: docker://opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v2
       options: --user 1001
     env:
+      START_CMD: ${{ matrix.config == 'query_enhanced' &&
```

Comment:
i really like this concept of `config` because it actually follows the format well with how OSD functional tests are structured!

only nit I have here is that this actually is a little bit hard to read. and any fixes or additional updates to this will require us to know to update both places.

so i think minimum we should consider the logic being where we append extra configurations if the matrix config is query enhanced instead of having two.

if that is not good enough then it might be time to start sourcing a .env config file that stores this information so that we can modify this stuff and make this file more approachable.